rtgui/edit.cc

```diff
@@ -791,24 +1078,29 @@ void EditSubscriber::switchOffEditMode()
     unsubscribe();
 }
 
-EditUniqueID EditSubscriber::getEditID()
+const EditUniqueID EditSubscriber::getEditID()
```

Comment:
The `const` before the return type should be superfluous for integral types as they are always immutable as rvalues, e.g. when they are returned from a function. I think GCC even warns about this at `-Wall`?